apps/ledger-live-desktop/tests/specs/speculos/receive.address.spec.ts

```diff
@@ -17,18 +18,18 @@ const accounts: Account[] = [
   Account.XTZ_1,
 ];
 
-//Reactivate test after fixing the GetAppAndVersion issue - Jira: LIVE-12581
+//Warning 🚨: test can fail due to GetAppAndVersion issue - Jira: LIVE-12581
 for (const [i, account] of accounts.entries()) {
-  test.describe.skip("Receive @smoke", () => {
+  test.describe.parallel("Receive @smoke", () => {
```

Comment:
why did you reactivate ? bug is still open 
@tdommes-ledger we'd need traction for this one : the more scenarios we'll automate with speculos, the more of them will be deactivated due to this issue (hence to be performed manually 
with the arrival of Flex and Stax devices, I'm pretty sure that if it's not a huge work to fix, [this](https://ledgerhq.atlassian.net/browse/LIVE-12581) should be treated in upcoming sprint and priority should be reviewed